src/main/java/com/google/sps/servlets/PerspectiveRequest.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.servlets;
+ 
+import com.google.common.collect.ImmutableMap;
+ 
+/**
+ * This class represents the data needed to send a request
+ * to the Perspective API and will be used to turn the data
+ * into a JSON formatted string.
+ */
+public class PerspectiveRequest {
+    private ImmutableMap<String, String> comment;
```

Comment:
Same here, is this map going to contain multiple entries? If not, use a pair.

Reply:
On second thought, can you clarify why  the comment needs to be stored together with "text"? As opposed to having a `COMMENT_TYPE = "text"` variable and a `String comment` variable

Reply:
it's for an API request, so I basically only created this class so that I could use Gson to convert it to JSON, and this is the way the API wants the request to be formatted. I tried nesting the maps myself, but it kept giving me an error, so I just let Gson do it from an object